modules/statistics/RTPStatsCollector.js

```diff
@@ -291,31 +291,52 @@ StatsCollector.prototype.errorCallback = function(error) {
  */
 StatsCollector.prototype.start = function(startAudioLevelStats) {
     if (startAudioLevelStats) {
+        const receiverStatsSupported = typeof window.RTCRtpReceiver !== 'undefined'
```

Comment:
can we have this defined in the `BrowserCapabilities` class as a method, so that others won't be duplicating it in future?

Reply:
sure